### DISCUSSION TITLE
Dev pathway on new button (react-button) vs old button

### DISCUSSION THREAD

**Discussion by imdavidmin** [microsoft/fluentui#17949] [General] [answered]
Hi,

Checking in with the team here after not having touched this area for a while for my project - I can see that the original plans to put the react-button into v8 has been pushed back, but the react-button is still in active development. Is the overall plan still to move towards the new button at some point, with the way to customise buttons the way it's done in react-button instead of v7 button?

My project isn't expected to be production / live until nearer to the end of 2021, so I thought I'd do the dev work now on the new button instead of having to do migration work later. It would help to know what's the overall direction is on the button and a very vague timeline, to see if what I'm planning makes sense.

Thanks!

@ecraig12345 @khmakoto

**Comment by parhamkmp** [comment]
###

**Comment by khmakoto** [comment, marked as answer]
Hi @imdavidmin, sorry for taking so much time to reply, this kind of went under my radar.

As you said, the plan to put `@fluentui/react-button` in v8 was pushed back and we're now targeting a v9 release. The plan is indeed still to move towards the new `Button` in `@fluentui/react-button` at some point.

As far as timelines are concerned I think @paulgildea or @gouttierre can help with that.
